src/cmdstan/command.hpp

```diff
@@ -702,14 +691,14 @@ int command(int argc, const char *argv[]) {
 
     std::vector<std::string> p_names;
     model.constrained_param_names(p_names, false, false);
-    for (size_t i = 1; i < p_names.size(); i++) {
+    for (size_t i = 1; i < p_names.size(); ++i) {
       output_stream << "g_" << p_names[i] << ",";
     }
     output_stream << "g_" << p_names.back() << "\n";
```

Comment:
[question]
Doesn't this skip the first entry and then output the last entry in `p_names` twice?  Is that intentional?  Some of this unexpected code could use some local doc.  For instance, why are we starting `i` at 1 rather than 0?